lib/attributor/types/model.rb

```diff
@@ -32,6 +32,7 @@ def self.inherited(klass)
 
       v = self.value_type
       va = self.value_attribute
+      re = self.requirements
```

Comment:
Should clone this so subclasses can add to just themselves. 

Although we don't inherit keys/attributes as it is, so maybe we don't inherit these at all?